source/production/F0.Minesweeper.Components/Cell.razor.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Threading.Tasks;
+using F0.Minesweeper.Components.Abstractions;
+using Microsoft.AspNetCore.Components;
+
+namespace F0.Minesweeper.Components
+{
+	public partial class Cell
+	{
+		[Parameter]
+		public Location Location { get; set; }
+
+		protected override void OnParametersSet()
+		{
+			if (Location == null) 
+			{
+				throw new ArgumentNullException(nameof(Location));
```

Comment:
hmmm ... I _do_ like the null-check ...
but is throwing `ArgumentNullException` "correct"?
Is there something like / should we introduce something like a `ParameterNullException`?
Or would this be total overkill and `ArgumentNullException` is perfectly fine and understandable?

Reply:
nvm ... on second thought, I like it!